server/lists/index.js

```diff
@@ -541,6 +541,12 @@ module.exports = {
             endpoint: '/case/document/reference/${caseId}/',
             adapter: documentListAdapter
         },
+        CASE_DOCUMENT_LIST_FINAL_RESPONSE: {
+            client: 'CASEWORK',
+            endpoint: '/case/document/reference/${caseId}/?type=Final%20response',
+            adapter: documentListAdapter
+        },
+        // todo: this should be removed and updated to use the non tenant specific option above.
```

Comment:
Did you mean to action this in this ticket, or is this for a future ticket?

Reply:
For a future ticket... if I remove it now I'll have to change stuff that's out of scope on this one.